### DISCUSSION TITLE
news about Orchestra ? Will Orchestra be platform independent ?

### DISCUSSION THREAD

**Discussion by racklking** [jhuckaby/Cronicle#568] [Q&A]
Hello all,

Sorry I am not a techie. I read the description, it says something about POSIX only.
My question is, if it is possible to run Cronicle on Windows by doing some tuning or including libraries etc. ?

**Comment by racklking** [comment]
sorry I beg my pardon, I got the answer under "Issues" section.
Since I cannot delete this topic, let me ask another way round:
What is the news about Orchestra ? Will Orchestra be platform independent ?

**Comment by mikeTWC1984** [comment]
Actually, cronicle is not that far from that. App and storage engines can run on Windows now (if you replace symlinks with file copies). The part that is crashing is launching event plugins. I did some analysis on that recently, looks like besides a few minor things the main problem is with launching scripts as executable, which works differently in Windows but should be trivial to patch. I was going to play around with it sometime soon, I'll post the result (if any) here
